library/build.gradle

```diff
@@ -55,8 +55,9 @@ configurations {
 
 dependencies {
     implementation 'com.android.support:support-annotations:' + rootProject.supportLibVersion
+    provided 'com.android.support:support-fragment:' + rootProject.supportLibVersion
```

Comment:
`provided` has been deprecated, `compileOnly` should be used instead [https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration?utm_source=android-studio#new_configurations](https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration?utm_source=android-studio#new_configurations)